Lib/test/test_binascii.py

```diff
@@ -198,6 +198,7 @@ def test_hex(self):
         self.assertEqual(s, u)
         self.assertRaises(binascii.Error, binascii.a2b_hex, t[:-1])
         self.assertRaises(binascii.Error, binascii.a2b_hex, t[:-1] + b'q')
+        self.assertRaises(binascii.Error, binascii.a2b_hex, bytes([255, 255]))
```

Comment:
Add tests for `b'0G'`, `b'G0'`, `b'0g'` and `b'g0'`.

Reply:
> _PyLong_DigitValue could be used also in binascii.a2b_qp

Shouldn't this be done in another PR?

Reply:
It's on to you. Since your initial code shared the table with `binascii.a2b_qp` I thought that you perhaps want to keep the relation between ` binascii.a2b_hex` and `binascii.a2b_qp`.

It is better to not squish intermediate commits. Squishing them makes reviewing harder.

Reply:
@serhiy-storchaka, sorry I removed my previous comment. I'll modify PR to use `_PyLong_DigitValue` in `binascii.a2b_qp` a bit later.